website/blog/2024-06-24-AltModels-Classes/index.mdx

```diff
@@ -0,0 +1,392 @@
+
+user_proxy (to functionbot):
+
+***** Response from calling tool (toolu_016wBUKVX2TToBaMqmiGvhav) *****
+New York will be 11 degrees fahrenheit
+***********************************************************************
+
+--------------------------------------------------------------------------------
+user_proxy (to functionbot):
+
+***** Response from calling tool (toolu_01Nxjeew2BspfKdZ85on3XkP) *****
+913.0000000000001 USD
+***********************************************************************
+
+--------------------------------------------------------------------------------
+functionbot (to user_proxy):
+
+Thank you for providing the information. I can now give your customer a
+summary of the weather in New York and the currency exchange from EUR to USD.
+
+Weather in New York:
+The current forecast for New York indicates that it will be 11 degrees
+Fahrenheit. This is quite cold, so your customer should be prepared with
+warm clothing for their trip.
+
+Currency Exchange:
+For 830 EUR, your customer will receive approximately 913 USD. This is based
+on the current exchange rate provided by our currency calculator.
+
+To summarize:
+1. Weather in New York: 11°F (very cold)
+2. Currency exchange: 830 EUR = 913 USD
+
+Your customer should pack warm clothes for the cold weather in New York and
+can expect to have about 913 USD for their trip after exchanging 830 EUR.
+
+TERMINATE
+
+--------------------------------------------------------------------------------
+Here's the LLM summary of the conversation:
+
+Certainly. I'll provide an email response to the customer with the travel
+information as requested.
+
+Dear Customer,
+
+We are pleased to provide you with the following information for your
+upcoming trip to New York:
+
+Weather Forecast:
+The current forecast for New York indicates a temperature of 11 degrees
+Fahrenheit. Please be prepared for very cold weather and pack appropriate
+warm clothing.
+
+Currency Exchange:
+We have calculated the currency exchange for you. Your 830 EUR will be
+equivalent to approximately 913 USD at the current exchange rate.
+
+We hope this information helps you prepare for your trip to New York. Have
+a safe and enjoyable journey!
+
+Best regards,
+Travel Assistance Team
+```
+
+So we can see how Anthropic's Sonnet is able to suggest multiple tools in a single response, with AutoGen executing them both and providing the results back to Sonnet. Sonnet then finishes with a nice email summary that can be the basis for continued real-life conversation with the customer.
+
+## More tips and tricks
+
+For an interesting chess game between Anthropic's Sonnet and Mistral's Mixtral, we've put together a sample notebook that highlights some of the tips and tricks for working with non-OpenAI LLMs. [See the notebook here](https://microsoft.github.io/autogen/docs/notebooks/agentchat_nested_chats_chess_altmodels).
+
+
+### Acknowledgements
+
+This blogpost has been revised with the help of ...
```

Comment:
@marklysze if there are none, we can remove this section.

Reply:
Thanks @Hk669, I left it in there while it gets reviewed and anyone assisting with the review can go in there (as per the last blog), but if no changes required then, yep, we can remove :).

Reply:
Okay, I've removed for now... will add it in if anyone has edits.

Reply:
The section is not removed yet.

Reply:
Ah, had it ready but didn't fully commit. Done now and I'll bring it up to main as well.

Reply:
Thanks, all done.